crates/rolldown_utils/src/path_ext.rs

```diff
@@ -44,6 +44,11 @@ impl PathExt for std::path::Path {
       _ => file_name,
     };
 
+    // Avoid file name contained an unexpected NUL byte.
```

Comment:
Should we do it in the begainning? I saw people return `\0[abslolute path]` in `resolve_id` hook.

Reply:
If normalized it at module loader resolve pharse, it maybe caused `debug_id` is incorrect.

Reply:
I mean should we put this logic ahead of this function.